hack/ldapsearch/README.md

```diff
@@ -0,0 +1,17 @@
+# ldapsearch.sh
+
+Usage
```

Comment:
Should we describe what it does here?  Else someone has to go read the code to find out.

Reply:
Sounds good. See next push.